airbyte-integrations/connectors/source-shopify/integration_tests/abnormal_state.json

```diff
@@ -1,44 +1,44 @@
 {
   "customers": {
-    "updated_at": "2020-06-08T21:09:26-07:00"
+    "updated_at": "2021-07-19T06:41:50-07:00"
   },
   "orders": {
-    "updated_at": "2020-08-03T14:42:06-07:00"
+    "updated_at": "2021-07-19T06:52:06-07:00"
   },
   "draft_orders": {
-    "updated_at": "2020-06-08T21:09:25-07:00"
+    "updated_at": "2021-07-07T08:18:59-07:00"
   },
   "products": {
-    "updated_at": "2021-06-25T06:13:37-07:00"
+    "updated_at": "2021-07-19T06:56:06-07:00"
```

Comment:
should we put this very very far into the future? 2024? feels like we update this every time we make a shopify change

Reply:
Updated.